documents/motions/08.2022.1 - Delegates.md

```diff
@@ -27,11 +28,11 @@ WCA Delegate is a role defined in the WCA Regulations to oversee official WCA Co
    7. WCA Delegates shall accept and comply with decisions of the WCA Board, WCA Officers, and WCA Teams/Committees.
    8. For every WCA Competition that they are overseeing, WCA Delegates:
       1. Represent the WCA.
-      2. Make sure Competition Organizers and Competition Staff are fully capable, equipped, and prepared to make the competition successful.
-      3. Make sure the competition is run according to the Mission, Spirit, Policies, and Regulations of the WCA.
+      2. Ensure Competition Organizers and Competition Staff are fully capable, equipped, and prepared to make the competition successful.
+      3. Ensure the competition is run according to the Mission, Spirit, Policies, and Regulations of the WCA.
       4. Timely submit a report about the competition to the community of WCA Delegates.
       5. Timely send in the results for the competition to the WCA Results Team.
```

Comment:
Not sure if the two above lines read the best and could do with a rewording

Reply:
Ensure timely submission of a report about the competition to the WCA Delegates
Ensure timely submission of the results for the competition to the WCA Results Team.